docs/setup_vscode.md

```diff
@@ -412,45 +386,56 @@ Edit the `"environment"` property in your `launch.json`.  If there's already an
     }
   ]
 ```
+{: data-highlight="3-4" }
 
 When ASan detects an error, VSCode will stop so that you can see the stack trace and inspect the current state of the program.  This configuration also turns off leak-checking (LSan), which can't run simultaneously with the visual debugger. If you do want to check for leaks, just run from the terminal with sanitizers enabled.
 
 If you're debugging something else in your program and don't want it to terminate on ASAN errors, you can change to `abort_on_error=0`.
 
-## Input redirection
-Skip this subsection on your first time through the tutorial.  You can use input redirection to avoid typing program input each time you run (for debugging) a program.
+### Input redirection
+<div class="primer-spec-callout info" markdown="1">
+Skip this subsection your first time through the tutorial.  You can come back to it.
+</div>
+
+You can use input redirection to avoid typing program input each time you run a program.  Here's an example program.
+```c++
+#include <iostream>
+#include <string>
+using namespace std;
+
+int main() {
+  cout << "What's your name?" << endl;
+  string name;
+  cin >> name;
+  cout << "Hello " << name << "!\n";
+}
+```
 
-Without input redirection, here's how you type input at the command line.  Notice that the program asks the user to `enter a filename` and then the user types `main_test_data.tsv`.  Then, the program asks the user to `enter a column name` and the user types `B`.
+Without input redirection, the user types input at the command line (highlighted).
 ```console
-$ make clean
-rm -rvf *.exe *~ *.out *.dSYM *.stackdump
 $ make main.exe
-g++ -Wall -Werror -pedantic -g --std=c++11 main.cpp stats.cpp p1_library.cpp -o main.exe
 $ ./main.exe
-enter a filename
-main_test_data.tsv
-enter a column name
-B
-...
+What's your name?
+Drew
+Hello Drew!
 ```
+{: data-highlight="4" }
 
-If we put the user input in a file we can automate the user input.  We'll put it in a file called `main_test.in`.
-```console
-$ cat main_test.in   # Peek at the contents of a file
-main_test_data.tsv
-B
-$ ./main.exe < main_test.in  # Redirect file content to main's stdin (cin)
-enter a filename
-enter a column name
-reading column B from main_test_data.tsv
-...
+Automate user input by putting it in a file.
 ```
+Drew
+```
+{: data-title="main_test.in" data-highlight="1" }
 
-Without input redirection, here's how to type input in the Visual Studio Code command line.  In some configurations, a window will pop up, in others, you'll type into a pane on the VS Code interface.
-
-<img src="images/vscode075.png" width="768px" />
+Redirect file `main_test.in` to stdin of `main.exe`.
+```console
+$ ./main.exe < main_test.in
+What's your name?
+Hello Drew!
+```
+{: data-highlight="1" }
 
-### Windows `launch.json` changes
+#### Windows `launch.json` changes
```

Comment:
No directions for CAEN Linux. After testing, it seems that the Windows changes work for CAEN Linux too.

Reply:
Good catch.  #51